pogobuf/pogobuf.client.js

```diff
@@ -1107,7 +1108,17 @@ function Client(options) {
                             api_url: responseEnvelope.api_url
                         });
 
-                        resolve(self.callRPC(requests, envelope));
+                        signedEnvelope.platform_requests = [];
+                        resolve(self.callRPC(requests, signedEnvelope));
+                        return;
+                    }
+
+                    /* Throttling, retry same request later */
+                    if (responseEnvelope.status_code === 52) {
+                        signedEnvelope.platform_requests = [];
+                        Promise.delay(2000).then(() => {
```

Comment:
Not a fan of this TBH. Can this be rewritten to use the same retry logic we already use elsewhere? In any case it should be configurable in case someone wants to roll their own throttling.